hw/cxl/cxl-host.c

```diff
@@ -355,7 +356,8 @@ void cxl_hook_up_pxb_registers(PCIBus *bus, CXLState *state, Error **errp)
 {
     /* Walk the pci busses looking for pxb busses to hook up */
     if (bus) {
-        QLIST_FOREACH(bus, &bus->child, sibling) {
+        QLIST_FOREACH(bus, &bus->child, sibling)
+        {
```

Comment:
Did clang-format do this on its own? Move curly to the above line.

Reply:
It was automatically done by clang-format.